docs/plugins/repository-s3.asciidoc

```diff
@@ -53,17 +53,27 @@ PUT _snapshot/my_s3_repository
 // CONSOLE
 // TEST[skip:we don't have s3 setup while testing this]
 
-Some settings are sensitive and must be stored in the {ref}/secure-settings.html[elasticsearch keystore].
-For example, to use explicit AWS access keys:
+Most client settings are specified inside `elasticsearch.yml`, but some are
+sensitive and must be stored in the {ref}/secure-settings.html[elasticsearch keystore].
+For example, to use explicit AWS access keys, execute:
 
 [source,sh]
 ----
 bin/elasticsearch-keystore add s3.client.default.access_key
 bin/elasticsearch-keystore add s3.client.default.secret_key
 ----
-
-The following are the available client settings. Those that must be stored in the keystore
-are marked as `Secure`.
+to add settings value to the keystore, before starting up the node. Any existing `s3`
```

Comment:
I'd suggest moving this sentence fragment up above the code block.  e.g. Change to something like this: "For example, before you start the node, run these commands to add AWS access key settings to the keystore:"